app/services/publisher_promo_stats_fetcher.rb

```diff
@@ -13,7 +13,7 @@ def perform
     response = connection.get do |request|
       request.headers["Authorization"] = api_authorization_header
       request.headers["Content-Type"] = "application/json"
-      request.url("/api/1/promo/owners/#{@publisher.id}/statsByTime?promo_id=#{@promo_id}")
+      request.url("/api/1/promo/owners/#{@publisher.owner_identifier}/statsByTime?promo_id=#{@promo_id}")
```

Comment:
Since `owner_identifier` is of the format `"publishers#uuid:#{id}"`, won't the hash (`#`) cause an issue in a URL?

Reply:
@nvonpentz yeah, using `URI.escape` is the right answer if the full owner identifier is expected here.

Reply:
@dgeb Thanks!  I will update here as well as the other places I use `#{channel.details.channel_identifier}`